src/pages/home/report/ReportActionCompose/SuggestionMention.tsx

```diff
@@ -270,9 +271,25 @@ function SuggestionMention(
                 }
 
                 return true;
-            });
+            }) as Array<PersonalDetails & {weight: number}>; // at this point we are sure that the details are not null
```

Comment:
Please update this comment to explain _why_ that is the case. Also, I think we should put the comment above the line, per our style guide, no?